.github/workflows/publish.yml

```diff
@@ -12,10 +12,10 @@ jobs:
     name: Publish
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v2
```

Comment:
Should lead to faster checkouts https://github.com/actions/checkout/releases/tag/v2.0.0